### PR TITLE
Add more documentation on bundles to README

### PR DESCRIPTION
**Pull request by mttdffy**
BUNDLES-3

I don't have a clear enough vision on how to improve this further, so here it is. Let me know if you have any thoughts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ Features
 * Adds a custom product bundle creation workflow to the admin
 * Adds a `package` product template for the storefront that displays all the packaged products together while still allowing separate add to cart buttons for each product
 * Adds a `family` product template for the storefront that displays all the packaged products together and allows adding any or all bundled products to a cart at the same time
-* Adds a the concept of `kit` products, a product bundle that defines its own variants as specific combinations of bundled product SKUs, and displays as a single product through checkout.
+* Adds the concept of `kit` products, a product bundle that defines its own variants as specific combinations of bundled product SKUs, and displays as a single product through checkout.
+* adds the `defer_to_components` inventory policy which is used for kits to allow inventory to be determined by the SKUs which make up a particular kit variant.
```
Comment (bencrouse) [inline README.md:14]:
```suggestion
* Adds the `defer_to_components` inventory policy which is used for kits to allow inventory to be determined by the SKUs which make up a particular kit variant.
```

File: README.md [hunk 2]
```diff
@@ -30,22 +32,40 @@ cd path/to/application
...
 ```
 
-Product Bundle Types
+Product Bundles
 --------------------------------------------------------------------------------
 
-There are three types of product bundles provided through this plugin:
+A **product bundle** is a product that contains other products.
+
+For some bundles, this is simply a way to display related products together on a single product detail page. In this plugin, we call those **discrete bundles**, which always show each individual product within the bundle and allow any or all of the products to be added to cart.
```
Comment (bencrouse) [inline README.md:40]:
```suggestion
For some bundles, this is simply a way to display related products together on a single product detail page. In this plugin, we call those **discrete bundles**, which always show each individual product within the bundle and allow any or all of the products to be added to cart. Each individual product will show as a separate item in the cart and throughout checkout.
```

File: README.md [hunk 3]
```diff
@@ -30,22 +32,40 @@ cd path/to/application
...
+A **product bundle** is a product that contains other products.
+
+For some bundles, this is simply a way to display related products together on a single product detail page. In this plugin, we call those **discrete bundles**, which always show each individual product within the bundle and allow any or all of the products to be added to cart.  
+
+### Discrete Bundles
 
 **Package**
 
-A package product is the most basic product bundle type. A package displays bundled products on a single detail page where a customer can select options and add each individual product to their cart one at a time. Each product within a package product displays its own options, quantity, and add to cart button.
+A package product is the most basic product bundle type. A package displays bundled products on a single detail page where a customer can select options and add each individual product to their cart one at a time. Each product within a package product displays its own options, quantity, and add to cart button. The bundle itself is never added directly to cart, but rather serves as a container of those related products.
```
Comment (bencrouse) [inline README.md:46]:
```suggestion
A package product is the most basic product bundle type. A package displays bundled products on a single detail page where a customer can select options and add each individual product to their cart one at a time. Each product within a package product displays its own options, quantity, and add to cart button. The package product itself is never added directly to cart, but rather serves as a container of those related products.
```

File: README.md [hunk 4]
```diff
@@ -30,22 +32,40 @@ cd path/to/application
...
+
+To a customer, a kit is displayed just like a normal product with a single set of options to select before adding the kit to their cart. The plugin provides you the option to show the bundled products, SKUs, and quantities to the customer after they make a selection, or leave that information hidden.
+
+**Variant Components**
+
+A variant made for a kit will have many components. Components define which SKUs a variant represents from among the bundled products. Additionally a component allows you to define a quantity, which allows a single kit variant to represent multiples of a single SKU.
+
+To better visualize the relationship of kits and their bundled products, examine the following example:
+
+![Kit Diagram](kit_diagram.png)
```
Comment (bencrouse) [inline README.md:64]: Nice, this is very helpful.

**Comment by ebigler** [comment]
Looks good to me! thanks

**Review by ebigler — APPROVED** [review]
As I mentioned in the standup meeting, my only suggestion is maybe mentioning something about kits having price adjustments but otherwise this looks good.

File: README.md [hunk 5]
```diff
@@ -30,22 +32,40 @@ cd path/to/application
...
 
-A kit product defines its own set of variants, with each variant representing a specific combination of SKUs and quantities of the products bundled within it. To a customer, a kit can be displayed just like a normal product with a single set of options to select before adding the kit to their cart. The plugin provides you the option to show the bundled products to the customer after they make a selection, or leave that information hidden. A kit is added to the cart and handled like any other product through checkout. Once an order containing a kit is placed, the items within the kit are expanded out of the kit to allow for proper tracking of each item's status through fulfillment. Kits allow for unique pricing of each combination of bundled products that enables you to offer savings to customers for buying items together.
+The variants in the kit are represented to the customer like variants of a regular product from the detail page through checkout. Once an order containing a kit is placed, the items within the kit are expanded out of the kit to allow for proper tracking of each item's status through fulfillment.
```
Comment (tubbo) [inline README.md:68]:
```suggestion
The variants in the kit are represented to the customer like variants of a regular product from the detail page through checkout. Once an order containing a kit is placed, the items within the kit are expanded out of the kit as separate order items, to allow for individual tracking of each item's status through fulfillment.
```

**Review by tubbo — CHANGES_REQUESTED** [review]

**Review by bencrouse — APPROVED** [review]

**Pull request merged**